api/cpp/cmake/SlintMacro.cmake

```diff
@@ -48,8 +57,15 @@ function(SLINT_TARGET_SOURCES target)
         set(scale_factor_target_prop "$<TARGET_PROPERTY:${target},SLINT_SCALE_FACTOR>")
         set(scale_factor_arg "$<IF:$<STREQUAL:${scale_factor_target_prop},>,,--scale-factor=${scale_factor_target_prop}>")
 
+        if (compilation_units GREATER 0)
+            foreach(cpp_num RANGE 1 ${compilation_units})                
+                list(APPEND cpp_files "${CMAKE_CURRENT_BINARY_DIR}/slint_generated_${_SLINT_BASE_NAME}_${cpp_num}.cpp")
```

Comment:
idea: name it `foobar.slint.cpp`?
But your slint_generated_ is not bad either.

Reply:
Iprefer `slint_generated_` as a prefix. It mages it very clear that it is generated and was done by slint.